app/src/main/java/org/p2p/wallet/sell/interactor/SellTransactionViewDetailsMapper.kt

```diff
@@ -0,0 +1,26 @@
+package org.p2p.wallet.sell.interactor
+
+import android.content.res.Resources
+import org.p2p.core.utils.formatToken
+import org.p2p.core.utils.formatUsd
+import org.p2p.wallet.R
+import org.p2p.wallet.moonpay.model.SellTransaction
+import org.p2p.wallet.sell.ui.lock.SellTransactionViewDetails
+
+class SellTransactionViewDetailsMapper(private val resources: Resources) {
+    fun fromDomain(transactions: List<SellTransaction>): List<SellTransactionViewDetails> {
```

Comment:
reusable logic for mapping for HistoryPresenter and TokenHistoryPresenter, both need SellTransactionViewDetails